feature/platform/controllercard/tests/SetRequest_controll_card_switchover/README.md

```diff
@@ -0,0 +1,34 @@
+# gNMI-1.19 ConfigPush after Control Card switchover
+
+## Summary
+This test verifys if large config can be bushed via gNMI SetRequest within 2 minutes after Control Card switchover. 
+
+## Procedure
+
+* Prepare large OpenConfig config file to be pushed within single setRequest.
+  * 150 LAG interfaces w/ ip, ipv6 configuration
+  * 800 Ethernet interfaces as memeber s of LAG
+  * 28 IPv4 and 28 IPv6 BGP neighbours
+  * ISIS on all trunk/LAG ports
+* Initiate Control Card switchover using gNOI SwitchControlProcessorRequest 
+* Immedietly after reciving SwitchControlProcessorResponce for  gNOI switchover, but no later then 120 second after calling gNOI SwitchControlProcessorRequest, send gNMI setRequest with prepared large config.
+* wait 120 second
```

Comment:
What's the purpose of this wait timer?

Reply:
I have rewritten this part, so perhaps intenton is cleaner now. Basicly, we give 120s time after switchover for controller card to seattle . At lest to point when new ACTIVE can igest new config.

Reply:
The new updates look good to me.  @LimeHat if not clear, please unresolve this thread with your question.